Index: sudoku/UI/UIDemo.java
===================================================================
diff --git a/sudoku/UI/UIDemo.java b/sudoku/UI/UIDemo.java
deleted file mode 100644
--- a/sudoku/UI/UIDemo.java	(revision fb98281cb53bbbfae6f5e57f452513a8cf9a3b05)
+++ /dev/null	(revision fb98281cb53bbbfae6f5e57f452513a8cf9a3b05)
@@ -1,75 +0,0 @@
-package sudoku.UI;
-
-import javax.swing.*;
-import java.awt.*;
-import java.awt.event.*;
-import java.util.Date;
-
-public class UIDemo {
-    public static void main(String[] args) {
-        Frame frame = new Frame("G G Bond");
-
-        frame.setSize(400, 400);
-        frame.setVisible(true);
-        frame.setResizable(false);
-        frame.setLocationRelativeTo(null);
-        frame.addWindowListener(new WindowAdapter() {
-            @Override
-            public void windowClosing(WindowEvent e) {
-                System.exit(0);
-            }
-        });
-        frame.setLayout(null);
-
-
-
-        Label label = new Label("G G Bond is my Hero!");
-        label.setBounds(20,50,200,30);
-        frame.add(label);
-        label.setFont(new Font("SimSong",Font.BOLD,15));
-        label.setForeground(Color.WHITE);
-        label.setBackground(Color.BLACK);
-
-        TextField field = new TextField();
-        field.setBounds(20,80,200,30);
-        frame.add(field);
-        field.setEchoChar('G');
-
-        Checkbox checkbox = new Checkbox("小黑子");
-        checkbox.setBounds(20,110,200,30);
-        frame.add(checkbox);
-        TextArea area = new TextArea();
-        area.setBounds(240,50,150,120);
-        frame.add(area);
-
-        CheckboxGroup group = new CheckboxGroup();
-        Checkbox checkbox1 = new Checkbox("小黑子");
-        Checkbox checkbox2 = new Checkbox("真爱粉");
-        checkbox1.setBounds(20,170,200,30);
-        checkbox2.setBounds(20,200,200,30);
-        frame.add(checkbox1);
-        frame.add(checkbox2);
-        checkbox1.setCheckboxGroup(group);
-        checkbox2.setCheckboxGroup(group);
-
-
-        Button button = new Button("Agree!");
-        button.setBounds(30,140,100,30);
-        button.addActionListener(new ActionListener() {
-            @Override
-            public void actionPerformed(ActionEvent e) {
-                System.out.println("You are GG fans!");
-                String text = field.getText();
-                System.out.println("输入的文本是："+ text);
-                System.out.println(checkbox.getState());
-                System.out.println(group.getSelectedCheckbox().getLabel());
-            }
-        });
-        frame.add(button);
-
-
-        frame.setAlwaysOnTop(true);
-        frame.setCursor(new Cursor(Cursor.CROSSHAIR_CURSOR));
-
-    }
-}
Index: sudoku/UI/NewGameUI.java
===================================================================
diff --git a/sudoku/UI/NewGameUI.java b/sudoku/UI/NewGameUI.java
deleted file mode 100644
--- a/sudoku/UI/NewGameUI.java	(revision fb98281cb53bbbfae6f5e57f452513a8cf9a3b05)
+++ /dev/null	(revision fb98281cb53bbbfae6f5e57f452513a8cf9a3b05)
@@ -1,98 +0,0 @@
-package sudoku.UI;
-
-import sudoku.UI.GameUI;
-
-import javax.swing.*;
-import java.awt.*;
-import java.awt.event.*;
-
-public class NewGameUI {
-    JFrame frame;
-    JButton button1,button2,button3,button4;
-    JPanel panel;
-    JDialog dialog;
-    JLabel label;
-
-    public NewGameUI(){
-        frame = new JFrame("NewGame");
-        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-        frame.setSize(300,200);
-        frame.setResizable(false);
-        frame.setLocationRelativeTo(null);
-        frame.setAlwaysOnTop(true);
-
-
-
-
-
-        //新游戏or读取上次进度
-        button1 = new JButton("新游戏");
-        button2 = new JButton("继续游戏");
-        button1.setBounds(20, 20, 60, 40);
-        button2.setBounds(20, 50, 60, 40);
-        button1.addActionListener(new ActionListener() {
-            @Override
-            public void actionPerformed(ActionEvent e) {
-                frame.setVisible(false);
-                dialog.setVisible(true);
-            }
-        });
-
-
-
-        panel = new JPanel();
-        panel.setLayout(new GridLayout(2, 1));
-        panel.add(button1);
-        panel.add(button2);
-
-        //弹窗提示是否覆盖历史对局
-        dialog = new JDialog(frame,"提醒",true);
-        dialog.setSize(300,200);
-        dialog.setLocationRelativeTo(null);
-
-        label = new JLabel("新游戏将覆盖历史进程");
-        label.setBounds(85,30,180,20);
-
-        button3 = new JButton("确定");
-        button4 = new JButton("取消");
-        button3.setBounds(70,70,60,40);
-        button4.setBounds(170,70,60,40);
-
-        // 为确定按钮添加点击事件监听器
-        button3.addActionListener(new ActionListener() {
-            @Override
-            public void actionPerformed(ActionEvent e) {
-                dialog.dispose(); // 关闭对话框
-                GameUI gameUI = new GameUI();
-            }
-        });
-
-        // 为取消按钮添加点击事件监听器
-        button4.addActionListener(new ActionListener() {
-            @Override
-            public void actionPerformed(ActionEvent e) {
-                dialog.dispose(); // 关闭对话框
-                frame.setVisible(true);
-            }
-        });
-
-        dialog.addWindowListener(new WindowAdapter() {
-            public void windowClosing(WindowEvent e) {
-                dialog.dispose(); // 关闭对话框
-                frame.setVisible(true);
-            }
-        });
-
-        dialog.setLayout(null);
-        dialog.add(label);
-        dialog.add(button3);
-        dialog.add(button4);
-
-
-
-        frame.add(panel);
-        frame.setVisible(true);
-
-    }
-
-}
Index: sudoku/UI/Framework.java
===================================================================
diff --git a/sudoku/UI/Framework.java b/sudoku/UI/Framework.java
deleted file mode 100644
--- a/sudoku/UI/Framework.java	(revision fb98281cb53bbbfae6f5e57f452513a8cf9a3b05)
+++ /dev/null	(revision fb98281cb53bbbfae6f5e57f452513a8cf9a3b05)
@@ -1,17 +0,0 @@
-package sudoku.UI;
-
-import javax.swing.*;
-
-public class Framework {
-
-    public Framework() {
-
-        NewGameUI newGameUI = new NewGameUI();
-
-
-    }
-
-    public static void main(String[] args) {
-        SwingUtilities.invokeLater(Framework::new);
-    }
-}
Index: sudoku/UI/GameUI.java
===================================================================
diff --git a/sudoku/UI/GameUI.java b/sudoku/UI/GameUI.java
deleted file mode 100644
--- a/sudoku/UI/GameUI.java	(revision fb98281cb53bbbfae6f5e57f452513a8cf9a3b05)
+++ /dev/null	(revision fb98281cb53bbbfae6f5e57f452513a8cf9a3b05)
@@ -1,21 +0,0 @@
-package sudoku.UI;
-
-import javax.swing.*;
-import java.awt.*;
-
-public class
-GameUI {
-    JFrame frame;
-    public GameUI() {
-        frame = new JFrame("GGBond做数独");
-        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-        frame.setSize(800, 600);
-        frame.setAlwaysOnTop(true);
-        frame.setCursor(new Cursor(Cursor.CROSSHAIR_CURSOR));
-        frame.setResizable(false);
-        frame.setLocationRelativeTo(null);
-        //frame.setLayout(null);
-
-        frame.setVisible(true);
-    }
-}
